ci/common.sh

```diff
@@ -769,6 +769,12 @@ amb_get_image_tag() {
 	echo "$(amb_get_image $@)" | rev | cut -d ":" -f1 | rev
 }
 
+# get the repository part of the image in the Ambassador deployment
+# (ie, "quay.io/datawire/aes:1.1.0" -> quay.io/datawire/aes)
+amb_get_image_repository() {
+	echo "$(amb_get_image $@)" | cut -d ":" -f1
```

Comment:
I think this will be unreliable. `registry.local:5000/ambassador:latest` will return `registry.local`

Reply:
```suggestion
	echo "$(amb_get_image $@)" | rev | cut -d ":" -f2- | rev
```